ush/forecast_det.sh

```diff
@@ -41,7 +41,7 @@ FV3_GFS_det(){
   # determine if restart IC exists to continue from a previous forecast
   RERUN="NO"
   filecount=$(find "${RSTDIR_ATM:-/dev/null}" -type f | wc -l)
-  if [ ${CDUMP} = "gfs" -a ${rst_invt1} -gt 0 -a ${FHMAX} -gt ${rst_invt1} -a ${filecount} -gt 10 ]; then
+  if [[ ( ${CDUMP} = "gfs" || ( ${RUN} = "gefs" && ${CDATE_RST} = "" )) && ${rst_invt1} -gt 0 && ${FHMAX} -gt ${rst_invt1} && ${filecount} -gt 10 ]]; then
```

Comment:
Is there a way to use `$RUN` or `$CDUMP` consistently.
I think replacing `$CDUMP` with `$RUN` would work here.
What is `$CDATE_RST`?

Reply:
Because CDUMP can only be gfs or gdas in g-w, so I use RUN to determine this is for GEFS based on the exglobal_forecast.sh. 

For $CDATE_RST, is for RERUN=YES. See https://github.com/NOAA-EMC/global-workflow/blob/1a7f0746a78b04cf9e92ef49aca40a81d3d91f42/scripts/exglobal_fcst_nemsfv3gfs.sh#L158. I will do more research on it.

Reply:
CDATE_RST is to set the restart CDATE, especially for the  forecast_d16_35. Then forecast_d16_35 can run restart from FHMAXH instead of from FHMAXH-$restart_interval.